__tests__/app.test.js

```diff
@@ -34,4 +43,26 @@ describe('/api/topics', ()    =>  {
     }); 
 });
 
-
+describe('/api', ()    =>  {
+    describe('GET', ()   =>   {
+        it('Status code 200', () =>  {
+            return request(app).get("/api").expect(200);
+        });
+        it('should respond with an accurate JSON object with avaliable endpoints', ()   => {
+            return request(app)
+            .get("/api")
+            .expect(200)
+            .then((response) => {
+            expect(response.body.endpoint).toEqual(endPoint);
+            });
+        });
```

Comment:
seeing as your second test block asserts that a 200 status code should come back you can really bin the first test. I would consider just letting the status code be asserted alongside the body in future too